tests/PHPStan/Rules/Arrays/data/bug-7469.php

```diff
@@ -0,0 +1,43 @@
+<?php declare(strict_types = 1);
+
+namespace Bug7469;
+
+function doFoo() {
+	$line = file_get_contents('php://input');
+
+	$keys = [
+		'lastName',
+		'firstName',
+		'languages',
+		'phone',
+		'email',
+		'voiceExample',
+		'videoOnline',
+		'videoTvc',
+		'radio',
+		'birthDate',
+		'address',
+		'bankAccount',
+		'ic',
+		'invoicingAddress',
+		'invoicing', // to bool
+		'note',
+	];
+
+	$data = array_combine($keys, $line);
+	$data['languages'] = explode(',', $data['languages']);
```

Comment:
the reported error on this line is interessting:

```
1) PHPStan\Rules\Arrays\NonexistentOffsetInArrayDimFetchRuleTest::testBug7469
Failed asserting that two strings are identical.
--- Expected
+++ Actual
@@ @@
-'
+'28: Cannot access offset 'languages' on array<'address'|'bankAccount'|'birthDate'|'email'|'firstName'|'ic'|'invoicing'|'invoicingAddress'|'languages'|'lastName'|'note'|'phone'|'radio'|'videoOnline'|'videoTvc'|'voiceExample', mixed>|false.
+28: Cannot access offset 'languages' on array<'address'|'bankAccount'|'birthDate'|'email'|'firstName'|'ic'|'invoicing'|'invoicingAddress'|'languages'|'lastName'|'note'|'phone'|'radio'|'videoOnline'|'videoTvc'|'voiceExample', mixed>|false.
 '
```

https://github.com/phpstan/phpstan-src/runs/7703414883?check_suite_focus=true

Reply:
fixed with [4b54905](https://github.com/phpstan/phpstan-src/pull/1596/commits/4b54905a4ad8cc4586e339cf667c4c66ef6a716d)

the error appeared only in the test-suit, not on the reported snippet, because of the phpstan level.
added a IF-case to compensate the possible `false` return, which happens on php7 only

Reply:
made the test compatible with php7 and php8.

there is still a double reported error on line 29, which I guess is a leftover bug?

Reply:
Why would it be a bug? The offset is accessed two times on the same line.